performance/core/fs/fileSystem.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const ncp = require('ncp').ncp;
+const del = require('del');
+
+function fileSystem() {
+    return {
+        createFolder: (folder) => {
+            if(!fs.existsSync(folder))
+                fs.mkdirSync(folder);
+        },
+        writeFileAsync(fileName, data) {
```

Comment:
This method is synchronous (because you use writeFileSync that is synchronous) yet you write "Async" in the name... it's confusing, can we either use the asynchronous version or change the name to Sync?